patterns/search-big-heading.php

```diff
@@ -1,8 +1,8 @@
 <?php
 /**
- * Title: Search - Big heading and search form
+ * Title: Heading and search form
```

Comment:
👍 
Please rename the file and the slug to match the new improved name. It will make the file easier to find if the pattern needs to be updated.